sdk/test/IntegrationTests/AWSSDK.IntegrationTests.NetFramework.csproj

```diff
@@ -51,6 +51,7 @@
     <Reference Include="System.ComponentModel.DataAnnotations" />
     <Reference Include="System.IO.Compression" />
     <Reference Include="System.Web" />
+    <Reference Include="System.Net.Http"/>
```

Comment:
ive been trying to figure out why i need System.Net.Http here if i already have it defined in the S3 integration tests. if anyone knows let me know. without this added here i get

```

C:\codebuild\tmp\output\src17698009\src\aws-sdk-net\sdk\test\Services\S3\IntegrationTests\CreatePresignedPostTests.cs(303,35): error CS1069: The type name 'ByteArrayContent' could not be found in the namespace 'System.Net.Http'. This type has been forwarded to assembly 'System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' Consider adding a reference to that assembly. [C:\codebuild\tmp\output\src17698009\src\aws-sdk-net\sdk\test\IntegrationTests\AWSSDK.IntegrationTests.NetFramework.csproj]
```